modyn/supervisor/internal/pipeline_executor/pipeline_executor.py

```diff
@@ -719,98 +714,17 @@ def _evaluate_and_store_results(
         last_timestamp: int,
     ) -> None:
         """Evaluate the trained model and store the results."""
-        # pylint: disable=too-many-locals
```

Comment:
`_single_evaluation` and `_store_evaluation_results` are now combined into one function `EvaluationExecutor._single_evaluation`